aw_watcher_window/main.py

```diff
@@ -68,9 +68,7 @@ def heartbeat_loop(client, bucket_id, poll_time, exclude_title=False):
             current_window = get_current_window()
             logger.debug(current_window)
         except Exception as e:
-            logger.error("Exception thrown while trying to get active window: {}".format(e))
-            traceback.print_exc()
-            current_window = {"appname": "unknown", "title": "unknown"}
+            logging.exception("Exception thrown while trying to get active window")
```

Comment:
This is good, but should use the module logger.

```suggestion
            logger.exception("Exception thrown while trying to get active window")
```